make/autoconf/libraries.m4

```diff
@@ -75,7 +75,7 @@ AC_DEFUN_ONCE([LIB_DETERMINE_DEPENDENCIES],
   fi
 
   # Check if alsa is needed
-  if test "x$OPENJDK_TARGET_OS" = xlinux; then
+  if test "x$OPENJDK_TARGET_OS" = xlinux -o "x$OPENJDK_TARGET_OS_ENV" = xbsd.freebsd -o "x$OPENJDK_TARGET_OS_ENV" = xbsd.netbsd; then
```

Comment:
We have traditionally used the pattern `if test ... || test ... ;then` in the configure files, like you did above in 
`lib-std.m4`. While it does not matter too much, I think the less variation we have the better, to keep the files as consistent as possible.